### PR TITLE
Clarify deployment and configuration for the issuance protocol

### LINKED ISSUE #267: Protocol document should explain (optional) attester behavior
Split attesters and issuers are not required for the basic issuance protocol, but we should explain (in a sentence or so) that if you are using a split attester, the client sends the request to the attester, and the attester forwards the request to the issuer if it accepts the client.

### PR DESCRIPTION
**Pull request by chris-wood**
Closes #267

### CODE REVIEW

File: draft-ietf-privacypass-protocol.md [hunk 1]
```diff
@@ -103,6 +103,44 @@ Unless otherwise specified, this document encodes protocol messages in TLS
...
+  TokenChallenge ---> Attest ------->                      |
+                    | TokenRequest ------------------>     |
+                    |                            (evaluate)|
+      Token  <------+  <-------------------  TokenResponse |
+                    |--------------------------------------/
+~~~
+{: #fig-issuance title="Issuance Overview"}
+
+The TokenChallenge inputs The basic token issuance protocols described in this
+document can be interactive or non-interactive, and per-origin or cross-origin.
```
Comment (tfpauly) [inline draft-ietf-privacypass-protocol.md:129]: This sentence seems incomplete/typoed?

File: draft-ietf-privacypass-protocol.md [hunk 2]
```diff
@@ -103,6 +103,44 @@ Unless otherwise specified, this document encodes protocol messages in TLS
...
+({{AUTHSCHEME, Section 2.1}}) and use the issuance protocols to produce
+corresponding Token values ({{AUTHSCHEME, Section 2.2}}). The issuance protocol
+describes how Clients and Issuers interact to compute a token using a one-round
+protocol consisting of a TokenRequest from the Client and TokenResponse from
+the Issuer. This interaction is shown below.
+
+~~~ aasvg
+  Origin             Client        Attester          Issuer
+
+                    +--------------------------------------\
```
Comment (tfpauly) [inline draft-ietf-privacypass-protocol.md:119]: Do we want the fancy stuff Martin added?
```+-----------------------------------.```, etc?

File: draft-ietf-privacypass-protocol.md [hunk 3]
```diff
@@ -420,6 +447,13 @@ respectively, used to produce tokens as input to the protocol. See
 
 Clients provide the following as input to the issuance protocol:
 
+- Issuer Request URI: A URI to which token request messages are sent. This can
```
Comment (tfpauly) [inline draft-ietf-privacypass-protocol.md:450]: This is good, thanks.

File: draft-ietf-privacypass-protocol.md (hunk 1 above)
Comment (chris-wood) [reply draft-ietf-privacypass-protocol.md]: Yep, fixed.

File: draft-ietf-privacypass-protocol.md (hunk 2 above)
Comment (chris-wood) [reply draft-ietf-privacypass-protocol.md:119]: Oh, I must have missed that. Let me apply it. Stand by.

Comment (chris-wood) [reply draft-ietf-privacypass-protocol.md]: Fixed

File: draft-ietf-privacypass-protocol.md [hunk 4]
```diff
@@ -103,6 +103,44 @@ Unless otherwise specified, this document encodes protocol messages in TLS
 notation from {{!TLS13=RFC8446}}, Section 3. Moreover, all constants are in
 network byte order.
 
+# Protocol Overview
+
+The issuance protocols defined in this document emody the core of Privacy Pass.
```
Comment (tfpauly) [inline draft-ietf-privacypass-protocol.md:108]: What is `emody` supposed to be?

Comment (tfpauly) [reply draft-ietf-privacypass-protocol.md:108]: Ah:
```suggestion
The issuance protocols defined in this document embody the core of Privacy Pass.
```

**Pull request merged**